pwiz_tools/Shared/CommonUtil/SystemUtil/ProcessRunner.cs

```diff
@@ -215,6 +222,13 @@ public static bool GoodIfExitCodeIsZero(string stderr, int exitCode)
                 }
 
             }
+            catch (IOException ex)  // CONSIDER: Should we handle more types like WrapAndThrowException does?
+            {
+                if (sbOutput != null)

```

Comment:
I would probably make this:
`if (sbOutput != null && sbOutput.Length > 0)`
or
`if (sbOuput?.Length > 0)`
so that we don't bother wrapping the exception unless there's at least one character of captured output.

Reply:
After considering, I guess I see value in having "Output:" in the exception details to make it completely clear that Skyline was collecting output but didn't get any before the exception was thrown. Otherwise, I can imagine wondering if the code to collect output failed for some reason. So, I am going to leave this as is.